### PR TITLE
enable hive lock house keeper

### PR DESCRIPTION
**Pull request by githubjianli**
Added variable `ENABLE_HIVE_LOCK_HOUSE_KEEPER` to support hive lock house keeper.

### CODE REVIEW

**Review by alangr28 — APPROVED** [review]

**Review by etapiaromero — APPROVED** [review]

**Review by RickArt — DISMISSED** [review]

File: files/startup.sh [hunk 1]
```diff
@@ -189,6 +189,16 @@ if [ ! -z ${AWS_WEB_IDENTITY_TOKEN_FILE} ]; then
...
 
+# Enable Hive lock housekeeper
+if [ ! -z ${ENABLE_HIVE_LOCK_HOUSE_KEEPER} ]; then
+    update_property.py hive.txn.manager org.apache.hadoop.hive.ql.lockmgr.DbTxnManager /etc/hive/conf/hive-site.xml
+    update_property.py hive.support.concurrency true /etc/hive/conf/hive-site.xml
+    update_property.py hive.compactor.initiator.on true /etc/hive/conf/hive-site.xml
+    update_property.py hive.compactor.cleaner.on true /etc/hive/conf/hive-site.xml 
+    update_property.py hive.compactor.worker.threads 1 /etc/hive/conf/hive-site.xml
+    update_property.py hive.txn.strict.locking.mode false /etc/hive/conf/hive-site.xml
+    update_property.py hive.exec.dynamic.partition.mode nonstrict /etc/hive/conf/hive-site.xml
```
Comment (patduin) [inline files/startup.sh:200]: this is not needed I think

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). 
 
+## [3.0.13] - 2023-08-07
+### Added
+- Added variable `ENABLE_HIVE_LOCK_HOUSE_KEEPER` to support hive lock house keeper.
```
[the remark this answers is not in the document]
Comment (abhimanyugupta07) [reply CHANGELOG.md:8]: We can create an issue in this project and explain it there and then refer it from the CHANGELOG.

**Review by patduin — CHANGES_REQUESTED** [review]
In our test some properties weren't needed, please double check

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). 
 
+## [3.0.13] - TBD
```
Comment (abhimanyugupta07) [inline CHANGELOG.md:6]:
```suggestion
## [3.0.13] - 2023-08-09
```

**Review by abhimanyugupta07 — APPROVED** [review]

**Review by patduin — APPROVED** [review]

**Pull request merged**